webpack/assets/javascripts/foreman_tools.js

```diff
@@ -10,6 +10,22 @@ import { sprintf, translate as __ } from './react_app/common/I18n';
 
 import { showLoading, hideLoading } from './foreman_navigation';
 
+import store from './react_app/redux';
+import { openConfirmModal as coreOpenConfirmModal } from './react_app/components/ConfirmModal';
+import { noop } from './react_app/common/helpers';
+
+export function openConfirmModal({
```

Comment:
now it can be used anywhere, also in legacy code, e.g:
`tfm.tools.openConfirmModal({ label: 'test'})`

btw the original core action is wrapped in this file with `dispatch` so no need to add it.